python/paddle/fluid/tests/unittests/parallel_executor_test_base.py

```diff
@@ -30,11 +30,17 @@
 __all__ = ['TestParallelExecutorBase']
 
 
+class DeviceType:
+    CPU = 1
+    GPU = 2
+    XPU = 3
```

Comment:
As discussed offline, you can use the `DeviceType` exposed by pybind directly  in the next PR.

Reply:
Yes, thanks for your advice, I will fix it in next PR.